src/VisualStudio/Core/Def/Implementation/Options/LanguageSettingsPersister.cs

```diff
@@ -222,6 +230,11 @@ private static void SetValueForOption(IOption option, ref LANGPREFERENCES3 langu
             }
         }
 
+        public Task PrefetchAsync(CancellationToken cancellationToken)
```

Comment:
when we do we want prefetch to be called?

Reply:
it is called from package initialization

Reply:
Ok.  Then it seems odd that it's exposed publicly on IOptionsPersister.  If it's something meant only for that codepath, i would expect it to be done in a different manner.  For example, the package initializer could simply import some new options interface we have just for this purpose and it could say: "if you need to do anything during package initialization, now's the time".

Reply:
that's just extra work. if it is public interface yes. but it is an internal interface